packages/generator-one-app-module/generators/app/index.js

```diff
@@ -13,12 +13,17 @@
  */
 
 /* eslint-disable no-underscore-dangle -- yeoman private methods use underscores */
+const path = require('path');
 const Generator = require('yeoman-generator');
 const chalk = require('chalk');
 const _ = require('lodash');
 const helper = require('./promptValidations');
 const packagejs = require('../../package.json');
 
+_.extend(Generator.prototype, require('yeoman-generator/lib/actions/install'));
+
+const getTemplatePath = (templateName) => path.join(__dirname, 'templates', templateName);
```

Comment:
Yeoman recommends absolute paths for more deterministic results. Tests also failed without this change after updating to Yeoman 5